en/docs/concepts/compliance/gdpr.md

```diff
@@ -0,0 +1,226 @@
+# General Data Protection Regulation
+
+## What is GDPR and who is affected?
+
+The General Data Protection Regulation (GDPR) is a legal framework formalized by the European Union (EU) in 2016. This regulation came into effect from 25, May 2018, and affects any organization that processes Personally Identifiable Information (PII) of individuals who live in Europe. Organizations that fail to demonstrate GDPR compliance are
+subjected to financial penalties. This regulation is critical to comply with due to both the heavy penalties and the fact that non-compliance puts the organization's reputation and integrity at stake. 
+
+GDPR affects organizations that are located in places within the EU, organizations that are located in places where the EU laws are followed, or when the clients of these organizations are EU citizens. Apart from these, there are special cases that do not fall under any of the previous scenarios but need to comply with the regulation nevertheless. For more information on the fundamentals of this regulation, see [Defining a Winning GDPR Strategy Part 1 - Introduction to GDPR](https://wso2.com/library/article/2017/12/introduction-to-gdpr/).
+
+---
+
+## How is a specialized IAM solution better at complying with regulations?
+
+Generally, identity data are scattered over several systems within an organization. In order to reach GDPR compliance, it is required to review, redesign, and modify each of these systems. This is a maintenance overhead and consumes a significant portion of annual IT budget and requires a specialized set of skills for continuous review
+and modification process. Considering the amount of data available wiithin an organization and the intensity of adverse impact on the organization in case of failure to comply with GDPR, skilled staff working on a centralized, secure environment for data processing is vital. 
+
+A GDPR-compliant IAM solution ensures that all identity profiles are managed centrally and shares only required data with other systems in an on-demand manner through well-known security standards such as [SAML](TO DO: insert-concepts) and [OpenIDConnect](TO DO: insert-concepts). Any efficient IAM solution supports anonymization to remove PII data from datasets, and pseudonymization to set aritifical identifiers to uniquely identify the user, instead of their personal data. This mitigates the risk of exposing the personal data of individuals to compromised environments. 
```

Comment:
OpenIDConnect --> OpenID Connect

the concepts page already exists, we can link to it